src/validateSchema.js

```diff
@@ -8,20 +8,30 @@ const getValidationErrors = (schema, model) => Object.keys(schema).reduce((acc,
   const value = model[key]
   const rules = schema[key]
 
+  const renderError = (condition, fallback) => {
+    let err;
+    try {
+      err = rules.formatError({ key, value, condition, rules, schema, model });
+    } catch (e) {
+      err = rules.formatError || fallback;
```

Comment:
My intention was that `formatError` could be either a function that returns a string, or just a string. I included the try/catch as a simpler way to handle type checking without all the JavaScript `typeof` oddities. So the try/catch logic was:

1. try to blindly invoke `formatError` as a function, which would throw if it was not one
2. if it's not a function (or is one that throws), assume `formatError` is a string
3. if `formatError` is falsy, use the fallback message

Having had to explain that in words now, I realise calling it `formatError` was the wrong choice (I originally called it `error` but then changed it before submitting the PR).